suspect/processing/channel_combination.py

```diff
@@ -3,14 +3,19 @@
 
 def svd_weighting(data):
     p, _, v = numpy.linalg.svd(data, full_matrices=False)
+    
     channel_weights = p[:, 0].conjugate()
+    # normalize so they sum to 1
+    channel_weights = channel_weights/numpy.sum(numpy.abs(channel_weights))
 
     # try some basic phase correction
     # in our truncation of the SVD to rank 1, we know that v[0] is our FID
     # use the first point of it to phase the signal
     phase_shift = numpy.angle(v[0, 0])
+    
+    channel_weights_phase_corrected = channel_weights * numpy.exp(-1j * phase_shift)        
 
-    return channel_weights * numpy.exp(-1j * phase_shift) / numpy.sum(numpy.abs(channel_weights))
+    return channel_weights_phase_corrected, channel_weights, phase_shift
```

Comment:
OK, I'm fine with changing that. I realize that there is additional phase correction coming down the line. The only issue is that I like to pull my channel weights off of the unsuppressed H2O signal since the SNR is so much higher, and then apply them to the H2O suppressed signal. The phase correction for one doesn't really work for the other, so it ends up making the H2O suppressed signal's phase a bit worse in some cases. This will be easily corrected later on since it's only 0th order anyway, so it's not that big of a deal. I can change it back if that would be easier.